docs/services/topology.md

```diff
@@ -0,0 +1,170 @@
+Topology Service
+================
+
+This document contains information about the service that runs <https://topology.opensciencegrid.org> and <https://topology-itb.opensciencegrid.org>.
+
+The source code for the service is in <https://github.com/opensciencegrid/topology>, in the `src/` subdirectory.
+This repository also contains the public part of the data that gets served.
+
+
+Deployment
+----------
+
+Topology is a webapp run with Apache on the host `topology.opensciencegrid.org`.
+The same daemon runs both the production and the ITB instances.
+The host is a VM at Nebraska;
+for SSH access, contact Derek Weitzel or Brian Bockelman.
+
+
+### Installation
+
+These instructions assume an EL 7 host with the EPEL repositories available.
+The software will be installed into `/opt/topology`.
+The following steps should be done as root.
+
+1.  Install prerequisites:
+
+        :::console
+        # yum install python36 gridsite httpd mod_ssl
+
+1.  Clone the repository:
+
+        :::console
+        # cd /opt
+        # git clone https://github.com/opensciencegrid/topology
+
+1.  Set up the virtualenv in the clone:
+
+        :::console
+        # cd topology
+        # python36 -m venv venv
+        # . ./venv/bin/activate
+        # pip install -r requirements.txt
+
+1.  Configure httpd, mod_wsgi, and mod_gridsite.  The .so file for mod_wsgi is located in `/opt/topology/venv/lib/python3.6/site-packages/mod_wsgi/server/`.
+
+1.  Ensure all the files listed in the section below exist and have the proper permissions.
```

Comment:
Do these last two steps need to be in the installation section? Seems like you have separate sections for both of them